aws/table_aws_appstream_fleet.go

```diff
@@ -0,0 +1,281 @@
+				Type:        proto.ColumnType_INT,
+			},
+			{
+				Name:        "max_user_duration_in_seconds",
+				Description: "The maximum amount of time that a streaming session can remain active, in seconds. If users are still connected to a streaming instance five minutes before this limit is reached, they are prompted to save any open documents before being disconnected.",
+				Type:        proto.ColumnType_INT,
+			},
+			{
+				Name:        "platform",
+				Description: "The platform of the fleet.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "stream_view",
+				Description: "The AppStream 2.0 view that is displayed to your users when they stream from the fleet. When APP is specified, only the windows of applications opened by users display. When DESKTOP is specified, the standard desktop that is provided by the operating system displays. The default value is APP.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "compute_capacity_status",
+				Description: "The capacity status for the fleet.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "fleet_errors",
+				Description: "The fleet errors.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "session_script_s3_location",
+				Description: "The S3 location of the session scripts configuration zip file. This only applies to Elastic fleets.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "usb_device_filter_strings",
+				Description: "The USB device filter strings associated with the fleet.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "vpc_config",
+				Description: "The VPC configuration for the fleet.",
+				Type:        proto.ColumnType_JSON,
+			},
+
+			// Steampipe standard columns
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("DisplayName"),
+			},
+			{
+				Name:        "tags",
+				Description: resourceInterfaceDescription("tags"),
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getAppStreamFleetTags,
+				Transform:   transform.FromValue(),
+			},
+			{
+				Name:        "akas",
+				Description: resourceInterfaceDescription("akas"),
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Arn").Transform(transform.EnsureStringArray),
+			},
+		}),
+	}
+}
+
+//// LIST FUNCTION
+
+func listAppStreamFleets(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	logger := plugin.Logger(ctx)
+
+	// Create Session
+	svc, err := AppStreamClient(ctx, d)
+	if err != nil {
+		logger.Error("aws_appstream_fleet.listAppStreamFleets", "connection_error", err)
+		return nil, err
+	}
+
+	// Unsupported region check
+	if svc == nil {
+		return nil, nil
+	}
+
+	params := &appstream.DescribeFleetsInput{}
+
```

Comment:
When we pass name = '' we are getting this error >
```
operation error AppStream: DescribeFleets, https response error StatusCode: 400, RequestID: 7616647e-16ef-4424-a260-ed34c984ff0f, api error ValidationException: 1 validation error detected: Value '[]' at 'names' failed to satisfy constraint: Member must satisfy constraint: [Member must have length greater than or equal to 1]
	operation error AppStream: DescribeFleets, https response error StatusCode: 400, RequestID: 613cb25a-8735-4b61-a5b3-e80c2c59fa6f, api error ValidationException: 1 validation error detected: Value '[]' at 'names' failed to satisfy constraint: Member must satisfy constraint: [Member must have length greater than or equal to 1] (SQLSTATE HV000)
```

We can handle this by adding 
```	
fleetname := d.EqualsQuals["name"].GetStringValue()
	if fleetname == "" {
		return nil, nil
	}
```